pages/k8s/1.18/components.md

```diff
@@ -23,21 +23,23 @@ bundle_release: '1.18'
 source of the information - please file a bug rather than editing the text -->
 
 This page details the charms, snaps, images and other components
-which comprise the 1.18 release of Charmed Kubernetes.
+which comprise the 1.18 release of Charmed Kubernetes. This is primarily a
+reference page, detailing very specifically what this release of
+**Charmed Kubernetes** contains. For installation, please refer to the
+[quickstart](../quickstart) page.
 
 Other information about this release can be found on the following pages:
 
-<a class='p-button--brand' href='/kubernetes/docs/'> Documentation </a>
 <a class='p-button--brand' href='/kubernetes/docs/1.18/release-notes'>Release notes </a>
-<a class='p-button--brand' href='/kubernetes/docs/1.18/upgrading'>Upgrading </a>
+<a class='p-button--brand' href='/kubernetes/docs/1.18/upgrade'>Upgrading </a>
```

Comment:
Oh, one more thing... Is the `upgrading` page getting renamed?  As of now, `upgrading` works:

https://ubuntu.com/kubernetes/docs/1.18/upgrading

But `upgrade` doesn't:

https://ubuntu.com/kubernetes/docs/1.18/upgrade